server/conn.go

```diff
@@ -293,6 +294,9 @@ func (c *conn) serve(conn Conn) error {
 					Type: imap.StatusRespBad,
 					Info: err.Error(),
 				}
+			} else if strings.Contains(err.Error(), "use of closed network connection") {
```

Comment:
Hmm, not of fan of this...

Reply:
I was getting a lot of those errors when running the server tests.

I think it only happens because the tests close the socket directly.

Reply:
I dropped this commit.